changelog/+tasks-display.fixed.md

```diff
@@ -0,0 +1,3 @@
+* Verify the tasks related to the proposed changes view to show or hide the tasks accordion in the details view
```

Comment:
This feature is not release yet right?
if so, maybe we should update previous changelog if needed

Reply:
it's the goal of this PR, to hide the accordion if there is no relevant task to display in the proposed changes view